vue-app/src/components/Cart.vue

```diff
@@ -645,10 +639,11 @@ export default class Cart extends Vue {
     )
   }
 
-  get canRegisterWithBrightId(): boolean {
+  get isBrightIdRequired(): boolean {
     return (
       userRegistryType === UserRegistryType.BRIGHT_ID &&
-      !this.$store.state.currentUser?.isVerified
+      (!this.currentUser?.brightId.isVerified ||
+        !this.currentUser?.isRegistered)
```

Comment:
Basically, with this condition, we are displaying the "Verify with BrightID" message whenever the user is not registered or is not verified.

A different approach can be made here. We could display different messages for each condition. For example:
- is not verified: "Verify with BrightID"
- is not registered: "Get registered"

But both are going to redirect the user to the same url, `/verify`.

What do you think? should we display these two dif messages regardless that they have the same target?

Reply:
I think the same CTA for both is fine.